### ISSUE TITLE
Interest Page

### ISSUE THREAD

**Issue by Joel-Groves** [MufasaTheMusician/eatwithme#17]
-Group Interests into groups
-Add the interests of a person you match with into your lists of interests with a tally of how many times you've matched with someone with a particular interests

**Comment by ROXtiGerf** [comment]
Currently blocked by UI testing and wireframe.

**Comment by MufasaTheMusician** [comment]
Our thoughts so far from Week 5 meeting and tutorial 
![](https://i.imgur.com/fEt09ch.jpg)

**Comment by MufasaTheMusician** [comment]
![](https://i.imgur.com/9ppL9sQ.jpg)
![](https://i.imgur.com/Bgtt1G9.jpg)
![](https://i.imgur.com/2UNfRDl.jpg)
![](https://i.imgur.com/fQWnvsw.jpg)
![](https://i.imgur.com/EmI8VMX.jpg)

**Comment by MufasaTheMusician** [comment]
We've identified two options: AI based vector space and tree infrastructure. The pros and cons of each need to be weighed before deciding which to implement.

Given that there will need to be time taken to make the decision, we're moving this out of scope for the alpha release. Once the alpha is out, we'll prioritise implementing matching.
